packages/apidom-ls/src/config/common/schema/lint/index.ts

```diff
@@ -56,6 +57,7 @@ import thenNonIfLint from './then--non-if';
 import thenTypeLint from './then--type';
 import titleTypeLint from './title--type';
 import typeTypeLint from './type--type';
+import typeTypeLintOpenApi30 from './type--type-openapi-3-0';
```

Comment:
Just consistent naming with the rest of config
```suggestion
import typeTypeOpenAPI3_0Lint from './type--type-openapi-3-0';
```

Reply:
@char0n 
this doesn't seem to be liked by linter: 

 Variable name `typeTypeOpenAPI3_0Lint` must match one of the following formats: camelCase, PascalCase, UPPER_CASE

Reply:
Yep, we disable it inline as in https://github.com/swagger-api/apidom/blob/c169eb3c5b00d2b2c6ecad34acb3304da7b3043d/packages/apidom-ls/src/config/openapi/header/lint/content--allowed-fields-3-0.ts#L6

We use this convention for element names containing version numbers as well.